queenbee/schema/operator.py

```diff
@@ -109,11 +109,14 @@ class LocalRequirements(BaseModel):
     # NOTE: yaml conversion doesn't play well with Enum hence using a validator.
     @validator('platform')
     def check_platform(cls, v):
-        assert v.lower() in cls._valid_platforms, \
-            '{} is not a valid platform. Try one of these platforms:\n - {}'.format(
-                v, '\n - '.join(pl.title() for pl in cls._valid_platforms)
-            )
-        return v.lower()
+        v = list(map(lambda x: x.lower(), v))
```

Comment:
PEP8 👮 

```suggestion
        v = [x.lower() for x in v]
```

Reply:
PEP8 doesn't allow map functions?!

Reply:
lambda is the issue here but generally speaking I think list comprehension is much cleaner in this case.